t/01-basic.t

```diff
@@ -89,17 +90,6 @@ is_deeply(
     'Baz::test_apply returns expected list'
 );
 
-is(
-    ( List::AllUtils::first { $_ > 5 } ( 1, 2, 5, 22, 7 ) ),
```

Comment:
If this is no longer working then that's a problem.

The reason I tested this is that there are other modules, like `Test::Deep`, that export subs with names that overlap with subs from `List::AllUtils`. In that case, you have to call _something_ by it's FQ name. I don't think we should break that feature of `List::AllUtils`. Maybe an `AUTOLOAD` would do the trick?